### ISSUE TITLE
administration user

### ISSUE THREAD

**Issue by pcano65** [SDIS62/prevarisc#582]
Bonjour,

J'ai deux utilisateurs préventionniste qui ont "disparu" de la console d'admin, pourtant ils sont bien dans la base de donnée, et sur mon prevarisc-test (dont la dernière synchro date d'un mois) les 2 utilisateurs apparaissent bien. Il s'est donc passé qqchose entre temps qui les ont fait disparaitre.
Pouvez vous me dire qu'elle est la requête exécutée pour afficher la liste des utilisateurs ?

Merci.

**Comment by kdubuc** [comment]
Pouvez vous me donner les valeurs contenues dans la table utilisateurs (uniquement pour ceux qui ont disparu). SANS le mot de passe :)

**Comment by pcano65** [comment]
6   bonin   xxxxx   2015-10-22 15:56:54     1   null    10  2
8   duarte  xxxxx   2016-03-10 08:47:11     1   null    12  2

**Comment by kdubuc** [comment]
Dans utilisateursinformations, vous avez bien les lignes avec l'id 10 et 12 ?

**Comment by pcano65** [comment]
Tout à fait. J'avais vérifié avant d'ouvrir l'issue. Je n'ai rien trouvé d'incohérent en base.

**Comment by pcano65** [comment]
Pour info, ces 2 préventionnistes n'apparaissaient pas dans la console d'admin et on ne pouvait pas non plus les affecter à un ERP.
J'ai supprimé les données associées dans la table etablissementinformationspreventionniste et tout a refonctionné ... par contre, il faut remettre le preventionniste dans la fiche ERP.

Une idée de la source du problème ???
